### PR TITLE
PPD-1395: upgrade vcluster to v0.16.4

### PR DESCRIPTION
**Pull request by a-kad**
PPD-1395: upgrade vcluster to v0.16.4

### CODE REVIEW

**Review by TylerGillson — APPROVED** [review]
lgtm

File: pkg/constants/constants.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ import "os"
 
 var (
 	// DefaultVClusterVersion is the default version of the virtual cluster to use
-	DefaultVClusterVersion = "0.13.0"
+	DefaultVClusterVersion = "v0.16.4"
```
Comment (shubhamrajvanshi) [inline pkg/constants/constants.go:7]: Please check if Hubble needs to send updated version/tag for vcluster

Comment (TylerGillson) [reply pkg/constants/constants.go:7]: @shubhamrajvanshi currently Hubble doesn't send any version details. That's part of the problem. Once Hubble starts configuring the vcluster version properly we won't need to change these defaults. See the notes in the description for https://spectrocloud.atlassian.net/jira/software/c/projects/PPD/issues/PPD-1395.

Comment (TylerGillson) [reply pkg/constants/constants.go:7]: Also, @ahmed-kadri-tt adding the `v` here just results in the controller having to remove it. Would be better to leave it off.

Comment (shubhamrajvanshi) [reply pkg/constants/constants.go:7]: Ok, I thought that was taken care as part of support for multiple distributions/flavors

**Pull request merged**